ci-operator/step-registry/cucushift/e2e/cucushift-e2e-ref.yaml

```diff
@@ -26,6 +26,7 @@
           and not @destructive
           and not @flaky
           and not @inactive
+          and not @long-duration
```

Comment:
@liangxia this `and not` already get it excluded, removing will have it included, right?

Reply:
@jhou1  We want to exclude the long duration tests, so we add this line here.